src/components/types/ApiTeam.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * API types for team-related operations
+ */
+
+export interface TeamPostApiPayload {
+  name: string;
+}
+
+export interface TeamPostApiResponse {
```

Comment:
`apiFetch()` func return this data https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/6e94a8f33d1a70318666d76bafe8bddfde6f6dad/src/composables/useApi.ts#L170-L173

or 

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/6e94a8f33d1a70318666d76bafe8bddfde6f6dad/src/composables/useApi.ts#L179